spec/ttml1.xml

```diff
@@ -5059,6 +5059,9 @@ before edge, the second applies to the start and end edges, and the third applie
 to the after edge.
 If four <loc href="#style-value-length">&lt;length&gt;</loc> specifications are provided, then they apply to before, end,
 after, and start edges, respectively.</p>
+<note role="clarification">
+<p>Pecentage values are relative to the dimension of the region to which they apply. For example, if the before and after edges correspond to the top and bottom edges of the region then percentage values that apply to either of the two edges are relative to the height of the region.</p>
```

Comment:
Looks good, even if I did have to read it three times. Would it be helpful to mention writing mode dependency explicitly to put the reader's mind in the right place for understanding that before and after are not always top and bottom?